PINCache/PINCache.m

```diff
@@ -55,6 +55,23 @@ - (instancetype)initWithName:(NSString *)name rootPath:(NSString *)rootPath
     return self;
 }
 
+- (instancetype)initWithName:(NSString *)name rootPath:(NSString *)rootPath serializer:(PINDiskCacheSerializerBlock)serializer deserializer:(PINDiskCacheDeserializerBlock)deserializer
```

Comment:
Can you make `initWithName:rootPath:` call this method with nil serializer and deserializer?